webpack.config.node.js

```diff
@@ -81,7 +82,9 @@ const webpackConfig = {
 					plugins: [ [
 						path.join( __dirname, 'server', 'bundler', 'babel', 'babel-plugin-transform-wpcalypso-async' ),
 						{ async: false }
-					] ]
+					] ],
+					cacheDirectory: path.join( __dirname, './build/.babel-cache' ),
```

Comment:
also, the client cache is in ./babel-cache, not under build. would having client and server share work? or is the cache id different across the builds?